App.js

```diff
@@ -9,13 +10,12 @@ const App = () => {
     </View>
   );
 };
-export default Category;
+export default Schedule;
 const styles=StyleSheet.create({
   rectangle3 : {
     width: 375,
     height: 275,
     backgroundColor: '#fcb69f'
-    
   },
-  
+
 });
```

Comment:
- Format full code using VSCODE
https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode
(Install suitable extensions for development https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify  )